airflow/kubernetes/pod_generator_deprecated.py

```diff
@@ -0,0 +1,620 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@ashb @potiuk @kaxil @turbaszek With this class we can actually maintain the existing executor_config with a deprecation warning until 2.1 or 2.2. This should help ease the transition. WDYT?